cmd/kubeadm/app/cmd/upgrade/apply.go

```diff
@@ -91,7 +91,7 @@ func NewCmdApply(apf *applyPlanFlags) *cobra.Command {
 
 			// If the version is specified in config file, pick up that value.
 			if flags.cfgPath != "" {
-				glog.V(1).Infof("fetching configuration from file", flags.cfgPath)
+				glog.V(1).Infof("fetching configuration from file %s", flags.cfgPath)
```

Comment:
is this applying a newline (\n) automatically?

Reply:
Yes, it will apply a newline (\n) automatically.